tf/net.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+import argparse
+import gzip
+import os
+import numpy as np
+import lc0net_pb2 as pb
+
+LC0_MAJOR = 0
+LC0_MINOR = 14
```

Comment:
I would think for actual submission, it would probably make sense to bump this to 15 (and correspondingly bump version in lc0).